File: COM_SIC_Activaciones/DataPowerRest/HeaderResponse.vb
'INC000002161718 inicio
Imports System

Public Class HeaderResponse

    Private _consumer As String
    Private _pid As String
    Private _status As New status
    Private _timestamp As String
    Private _varArg As String

    Public Property consumer() As String
        Get
            Return _consumer
        End Get
        Set(ByVal Value As String)
            _consumer = Value
        End Set
    End Property

    Public Property pid() As String
        Get
            Return _pid
        End Get
        Set(ByVal Value As String)
            _pid = Value
        End Set
    End Property

    Public Property timestamp() As String
        Get
            Return _timestamp
        End Get
        Set(ByVal Value As String)
            _timestamp = Value
        End Set
    End Property
    Public Property varArg() As String
        Get
            Return _varArg
        End Get
        Set(ByVal Value As String)
            _varArg = Value
        End Set
    End Property

    Public Property status() As status
        Get
            Return _status
        End Get
        Set(ByVal Value As status)
            _status = Value
        End Set
    End Property



    Public Sub New()
        _status = New Status
    End Sub
End Class

File: COM_SIC_Entidades/DataPowerRest/HeadersGenerics/HeaderResponse.cs
//INICIO PROY-140332 (Try & Buy)
using System;
using System.Runtime.Serialization;

namespace COM_SIC_Entidades.DataPowerRest.HeadersGenerics
{
	/// <summary>
	/// Summary description for HeaderResponse.
	/// </summary>
	[Serializable]
	public class HeaderResponse
	{
		private string _consumer;
		private string _pid;
		private Status _status;
		private string _timestamp;
		private string _varArg;		

		public string consumer
		{
			set{_consumer= value;}
			get{ return _consumer;}
		}
		public string pid
		{
			set{_pid= value;}
			get{ return _pid;}
		}
		public Status status
		{
			set{_status= value;}
			get{ return _status;}
		}
		public string timestamp
		{
			set{_timestamp= value;}
			get{ return _timestamp;}
		}
		public string varArg
		{
			set{_varArg= value;}
			get{ return _varArg;}
		}

		public HeaderResponse()
		{
			_status = new Status();
		}
	}
	//FIN PROY-140332 (Try & Buy)
}
